sdks/python/apache_beam/runners/interactive/interactive_environment.py

```diff
@@ -171,7 +171,6 @@ def __init__(self):
     self._computed_pcolls = set()
     # Always watch __main__ module.
     self.watch('__main__')
-    self._is_py_version_ready = True
```

Comment:
version checking is removed at https://github.com/apache/beam/pull/14137/files#diff-494028baa234df312d977c601f13723366cb547e803fe96180375c81c5765b91L175